### ISSUE TITLE
guard-liverload not working on ruby 2+

### ISSUE THREAD

**Issue by zoras** [guard/guard-livereload#131]
```[ruby]
# Gemfile
 group :development do
   # Guard plugins
  gem 'guard-bundler'
  gem 'guard-rspec'
  gem 'guard-livereload', require: false
end
```

Console error:
```[bash]
❯ guard init livereload                                                                      ✱ ◼
14:19:53 - ERROR - Could not load 'guard/livereload' or'    ' find class Guard::Livereload
14:19:53 - ERROR - Error is: cannot load such file -- guard/guard
14:19:53 - ERROR - /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-livereload-1.4.0/lib/guard/livereload.rb:2:in `require'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-livereload-1.4.0/lib/guard/livereload.rb:2:in `<top (required)>'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/plugin_util.rb:105:in `require'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/plugin_util.rb:105:in `rescue in plugin_class'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/plugin_util.rb:97:in `plugin_class'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/plugin_util.rb:56:in `initialize_plugin'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/internals/plugins.rb:26:in `add'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/dsl.rb:185:in `block in guard'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/dsl.rb:182:in `each'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/dsl.rb:182:in `guard'
> [#] /Users/zoras/code/etherapi-p2/Guardfile:29:in `evaluate'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/dsl.rb:377:in `instance_eval'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/dsl.rb:377:in `evaluate'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/guardfile/evaluator.rb:90:in `evaluate'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/cli/environments/valid.rb:34:in `initialize_guardfile'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/cli.rb:174:in `init'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/aruba_adapter.rb:32:in `execute'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/aruba_adapter.rb:19:in `execute!'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/bin/guard:11:in `<top (required)>'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/bin/guard:23:in `load'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/bin/guard:23:in `<main>'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in `eval'
> [#] /Users/zoras/.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in `<main>'
/Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/dsl.rb:383:in `rescue in evaluate': Invalid Guardfile, original error is:

Could not load class: "Livereload",
backtrace:
	(dsl)> $GEM_PATH[0]/gems/guard-2.11.1/lib/guard/plugin_util.rb:57:in `initialize_plugin'
	(dsl)> $GEM_PATH[0]/gems/guard-2.11.1/lib/guard/internals/plugins.rb:26:in `add'
	(dsl)> $GEM_PATH[0]/gems/guard-2.11.1/lib/guard/dsl.rb:185:in `block in guard'
	(dsl)> $GEM_PATH[0]/gems/guard-2.11.1/lib/guard/dsl.rb:182:in `each'
	(dsl)> $GEM_PATH[0]/gems/guard-2.11.1/lib/guard/dsl.rb:182:in `guard'
	(dsl)> ./Guardfile:29:in `evaluate'
	(dsl)> $GEM_PATH[0]/gems/guard-2.11.1/lib/guard/dsl.rb:377:in `instance_eval'
	(dsl)> $GEM_PATH[0]/gems/guard-2.11.1/lib/guard/dsl.rb:377:in `evaluate'
	(dsl)> $GEM_PATH[0]/gems/guard-2.11.1/lib/guard/guardfile/evaluator.rb:90:in `evaluate'
	(dsl)> $GEM_PATH[0]/gems/guard-2.11.1/lib/guard/cli/environments/valid.rb:34:in `initialize_guardfile'
	(dsl)> $GEM_PATH[0]/gems/guard-2.11.1/lib/guard/cli.rb:174:in `init'
	(dsl)> $GEM_PATH[0]/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	(dsl)> $GEM_PATH[0]/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	(dsl)> $GEM_PATH[0]/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
	(dsl)> $GEM_PATH[0]/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	(dsl)> $GEM_PATH[0]/gems/guard-2.11.1/lib/guard/aruba_adapter.rb:32:in `execute'
	(dsl)> $GEM_PATH[0]/gems/guard-2.11.1/lib/guard/aruba_adapter.rb:19:in `execute!'
	(dsl)> $GEM_PATH[0]/gems/guard-2.11.1/bin/guard:11:in `<top (required)>'
	(dsl)> $GEM_PATH[0]/bin/guard:23:in `load'
	(dsl)> $GEM_PATH[0]/bin/guard:23:in `<main>'
	(dsl)> $GEM_PATH[0]/bin/ruby_executable_hooks:15:in `eval'
	(dsl)> $GEM_PATH[0]/bin/ruby_executable_hooks:15:in `<main>' (Guard::Dsl::Error)
	from /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/dsl.rb:377:in `evaluate'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/guardfile/evaluator.rb:90:in `evaluate'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/cli/environments/valid.rb:34:in `initialize_guardfile'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/cli.rb:174:in `init'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/aruba_adapter.rb:32:in `execute'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/lib/guard/aruba_adapter.rb:19:in `execute!'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/gems/guard-2.11.1/bin/guard:11:in `<top (required)>'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/bin/guard:23:in `load'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/bin/guard:23:in `<main>'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in `eval'
	from /Users/zoras/.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by e2** [comment]
Works for me (Linux) in MRI 2.2.0.

You are not using bundler to run Guard - does it work with `bundle exec`?

What's in `Gemfile.lock` (so I can compare for differences)?

Also, what does:  `echo $RUBYGEMS_GEMDEPS` say (in case you have a custom setting - RubyGems nowadays can read the Gemfile.lock, etc.)

**Comment by zoras** [comment]
The problem was fixed after updating guard-livereload to 2.4.0 (was 1.4.0) due to gem dependency issue. I was expecting it to be latest and didn't check on the gem version.

```[ruby]
Bundler could not find compatible versions for gem "multi_json":
  In Gemfile:
    guard-livereload (~> 2.0) x86-mingw32 depends on
      multi_json (~> 1.8) x86-mingw32

    eligible (>= 0) x86-mingw32 depends on
      multi_json (1.7.9)
```
